bin/crontabScripts/clearIndex.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#:Title: clearIndex.sh
+#:Author: Alan Coleman
+
+#clears content stored in "my_index"
+curl -XDELETE 'localhost:9200/my_index'
```

Comment:
Is there any way we can assert if the index has been removed?

Reply:
when that command is ran, a line comes up saying if it has been acknowledged. if acknowledged =true, it was successful, otherwise an error occurs

Reply:
In that case, it should be ok.